src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    width: 166px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    line-height: 14px;
+    margin-left: 16px;
+  }
+
+  &__details {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 16px 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-accent;
+  }
+
+  &__price {
+    display: flex;
+    margin: 0 16px;
+    justify-content: space-between;
+    color: $secondary;
+  }
+
+  &__price-string {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__sum {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #FFF;
+    font-weight: 700;
+    line-height: 16px;
+    width: 166px;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead